common/confirm/confirm-dialog.html

```diff
@@ -1,18 +1,18 @@
-<div ng-class="{'in': $ctrl.modalVisible}" class="modal" tabindex="-1" role="dialog" aria-hidden="true">
+<div class="modal in" tabindex="-1" role="dialog" aria-hidden="true">
```

Comment:
I guess that "in" class is just a relict without any usage

Reply:
I think its needed so the modal doesn't 'open' in invisible state. Bootstrap things.